ckanext/etsin/mappers/iso_19139.py

```diff
@@ -7,21 +7,26 @@
 log = logging.getLogger(__name__)
 
 # Overwrites ckanext-spatial's get_package_dict
-# TODO: Not yet implemented
 def iso_19139_mapper(self, context, data_dict):
 
-    package_dict = data_dict['package_dict']
-
     log.info("Greetings from within the spatial mapper. Here's the dict I'm working with:")
-    log.info(package_dict)
-
-    # Let's assume that Metax expects a key named "metax-title".
-    # We can edit values given by ckanext-spatial's default get_package_dict
-    package_dict['metax-title'] = package_dict['title']
-    package_dict.pop('title', None)
-
-    # We can also derive values from ISO values, XML tree or harvest object
-    package_dict['metax-abstract'] = data_dict['iso_values']['abstract']
+    log.info(data_dict)
+
+    # Start with an empty slate
+    package_dict = []
+
+    # Metax obligatory fields
+    package_dict['urn_identifier'] = "todo" # We can't rely on harvested packages having a URN identifier
+    package_dict['preferred_identifier'] = "todo" # We can't rely on harvested packages having a URL/URN identifier
+    package_dict['modified'] = data_dict['metadata_modified_date']
+    package_dict['versionNotes'] = "" # We don't have any - is empty string ok?
+    package_dict['title'] = [{"TODO": data_dict['iso_values']['title']}] # What language codec is used here? We may have it in ['extras']['metadata-language'], but no guarantees.
+    package_dict['description'] = [{"TODO": data_dict['iso_values']['abstract']}] # Like above
+    package_dict['totalbytesize'] = 0 # We don't have this information - is 0 ok or should here be another code?
+    package_dict['ready_status'] = "Finished" # Can we always use "Finished" or do we have to use "Removed", too? When we call metax api delete, does it care about this value?
```

Comment:
We can use "Finished". The Removed status is no longer after I mentioned this and we figured it is not needed. So no need to set this field's value when removing dataset